File: submit.php
<?php

$First_name = $_POST['First_name'];
$Last_name = $_POST['Last_name'];


$host = "localhost";
$user = "root";
$password = "mynewpassword";
$db = "cust_details";

$conn = new mysqli($host, $username, $password, $db);

if ($conn->connect_error){
	die("Connection failed: ". $conn->connect_error);
}
		  
		  
$sql = "INSERT INTO customer (First_name, Last_name) VALUES ('$First_name', '$Last_name')";
         


if ($conn->query($sql) === TRUE) {
	echo "ADDED: ".$First_name.", ".$Last_name;
} else {
	echo "Error: ".$sql."<br>".$conn->error;
}

$conn->close();




?>

File: submit.py
#!/usr/bin/python3

print("Content-Type:text/html")
print()
import cgi

print ("<h1>Welcome</h1>


form=cgi.FieldStorage()

First_name=form.getvalue("First_name")
Last_name=form.getvalue("Last_name")



import mysql.connector


con=mysql.connector.connect(
  host="localhost",
  user="root",
  password="mynewpassword",
  database="cust_details"
)

cur=con.cursor()

cur.execute("INSERT INTO customer VALUES (%s, %s)", (First_name,Last_name))
con.commit()
cur.close()
con.close()


print("<h3>record inserted.</h3>")